package.json

```diff
@@ -8,19 +8,19 @@
     "classnames": "2.2.3",
     "codemirror": "^5.16.0",
     "codemirror-graphql": "^0.5.2",
-    "graphiql": "^0.7.3",
-    "graphql": "^0.6.0",
-    "graphql-relay": "0.4.2",
+    "graphiql": "^0.7.8",
```

Comment:
In GraphiQL [v0.7.5](https://github.com/graphql/graphiql/releases/tag/v0.7.5) I improved some of the things that I discovered as I implemented the toolbox version GraphiQL:

https://github.com/graphql/graphiql/issues/152

I would be great if you could check these 2 points:

* The "Run" shortcut (<kbd>Ctrl</kbd> + <kbd>Enter</kbd> or <kbd>Cmd</kbd> + <kbd>Enter</kbd>) is still working on different tabs after the update. (and executes query only on a selected tab)
* The GraphiQL styles look right (since the css selectors should be affected by the change)

If there is some work that needs to be done in this area we can either keep the version of GraphiQL unchanged here and then make these updates in a separate PR or we can fix this stuff in this one. WDYT?